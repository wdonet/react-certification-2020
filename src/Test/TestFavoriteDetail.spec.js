import React from 'react';
import FavoriteDetail from '../pages/FavoriteVideos/FavoriteVideoDetail.page';

describe('FavoriteDetail Components', () => {
  const VideoArray = [
    {
      id: {
        videoId: '123',
      },
      getVideoId: 'cHm3wVzcHjnE',
      snippet: {
        title: 'rick roll',
        thumbnails: {
          medium: {
            url: 'https://i.ytimg.com/vi/Po3VwR_NNGk/mqdefault.jpg',
          },
        },
      },
    },
  ];

  it('Take FavoriteDetail Snapshot ', () => {
    const component = <FavoriteDetail {...VideoArray} />;
    expect(component).toMatchSnapshot();
  });
});
