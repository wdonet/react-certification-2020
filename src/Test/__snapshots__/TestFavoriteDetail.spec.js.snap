// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoriteDetail Components Take FavoriteDetail Snapshot  1`] = `
<FavoriteVideoDetail
  0={
    Object {
      "getVideoId": "cHm3wVzcHjnE",
      "id": Object {
        "videoId": "123",
      },
      "snippet": Object {
        "thumbnails": Object {
          "medium": Object {
            "url": "https://i.ytimg.com/vi/Po3VwR_NNGk/mqdefault.jpg",
          },
        },
        "title": "rick roll",
      },
    }
  }
/>
`;
